Misc/NEWS.d/next/Documentation/2022-04-29-16-23-14.gh-issue-92047.oKYyuI.rst

```diff
@@ -0,0 +1,5 @@
+From https://docs.python.org/3.11/c-api/init.html?highlight=py_getversion#c.Py_GetVersion:

```

Comment:
Please remove this NEWS entry, it's not needed.

Reply:
Removed.